application/views/questionAdministration/subquestionRow.twig

```diff
@@ -25,7 +25,7 @@
 
         <!-- Code (title) -->
         <td class="code-title" style="vertical-align: middle;">
-            {{ subquestion.title }}
+            {{ subquestion.title|escape('html_attr') }}
```

Comment:
It's html here : `{{ subquestion.title|escape('html') }}` ;)

Reply:
done. Thanks